src/NServiceBus.Persistence.ServiceFabric/SagaPersister/RuntimeSagaInfo.cs

```diff
@@ -0,0 +1,54 @@
+﻿namespace NServiceBus.Persistence.ServiceFabric
+{
+    using System;
+    using System.IO;
+    using System.Text;
+    using Newtonsoft.Json;
+
+    class RuntimeSagaInfo
+    {
+        // ReSharper disable once NotAccessedField.Local
+        Type sagaDataType;
+        JsonSerializer jsonSerializer;
+        Func<TextReader, JsonReader> readerCreator;
+        Func<TextWriter, JsonWriter> writerCreator;
+
+        readonly Version CurrentVersion;
+
+        public RuntimeSagaInfo(Type sagaDataType,
```

Comment:
From the persistence perspective the saga data type has more importance than the saga type